sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -25,6 +25,12 @@
     'asia-east1',
 ]
 
+# Default instance family for CPU-only VMs.
+# General-purpose instance with Intel Ice Lake 8373C or Cascade Lake 6268CL
+# 4 GB RAM per 1 vCPU
+_DEFAULT_INSTANCE_FAMILY = 'n2'
```

Comment:
```suggestion
_DEFAULT_INSTANCE_FAMILY = 'n2-standard'
```
?

Reply:
I think either makes sense; it depends on how we define "instance family". I just tried to make it consistent with `_DEFAULT_HOST_VM_FAMILY = 'n1'`.

Reply:
Does GCP define this term?

If not (or even if it does), I'd prefer to spell it out at the constants' definitions, otherwise the reader is left wondering whether it's standard/highmem/.., and it's only revealed by poking deep into the implementation L177.

Reply:
> Does GCP define this term?

To my understanding, their official terminologies "machine series" (e.g., N1, N2, N2D) and "machine types" (e.g, n1-highmem-8). The "instance family" is our own terminology.

> If not (or even if it does), I'd prefer to spell it out at the constants' definitions, otherwise the reader is left wondering whether it's standard/highmem/.., and it's only revealed by poking deep into the implementation L177.

Alright. Fixed.